### ISSUE TITLE
Offline version that can run locally

### ISSUE THREAD

**Issue by ryales** [BrcMapsTeam/europe-15-situational-awareness#18]
Copy and modify the js file to be able to read from uploaded json. Doesn't have to be implemented over the HXL for the live site, but we need a way to run things offline if needed when in the field or if HXL server is down (like today)

**Comment by SimonbJohnson** [comment]
I don't think this is something we should maintain consistently as it will be a high workload.  We could once a x time period push up a presentation version to the amazon server and have a script to prep this.

**Comment by ryales** [comment]
Exactly. Totally agree. I wasn't looking for something that was always up to date -- was thinking more about having a version of the javascript library that can be used for running locally as an alternative. Where you can read from some uploaded csv/json files (with the right structure) on the off chance there's no access to the net or HXL -- I'll move down the priority and leave it as "pet project" :)

**Comment by SimonbJohnson** [comment]
I don't think we should be catering to offline so much as in Europe the use cases are rare.  In terms of robustness though we could potentially look into html5 manifests. I've not yet tried them myself, but will look into it

https://en.wikipedia.org/wiki/Cache_manifest_in_HTML5
